### DISCUSSION TITLE
what is the difference?

### DISCUSSION THREAD

**Discussion by elen07zz** [wkpark/uddetailer#74] [General]
Hello, i would like to know what is the difference between this extension and adetailer?

**Comment by wkpark** [comment]
basically, adetailer and uddetailer work exactly the same but the details are different. for example:
- uddetailer supports popular `mmdet` models, but adetailer does not.
- adetailer supports some macro/script like feature, but uddetailer does not.
- uddetailer only support 2-detection models, but addetailer support more than 2-models.
- uddetailer support simple inpainting helper.
- uddetailer support Load/Save settings.
- uddetailer supports object class designation.
and both have similar features but differently implemented
